packages/block-library/src/list/additional-settings.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { InspectorControls } from '@wordpress/block-editor';
+import {
+	TextControl,
+	PanelBody,
+	ToggleControl,
+} from '@wordpress/components';
+
+const AdditionalSettings = ( { setAttributes, ordered, reversed, start } ) =>
```

Comment:
Just a suggestion: perhaps call it `OrderedListSettings` and move the `ordered` check?

Reply:
I like that approach much better. Thanks for the suggestion!